poncho/src/poncho/package_serverize.py

```diff
@@ -112,10 +112,129 @@ def main():
 
                     break
             else:
-                print("Network function could not read from worker\n", file=sys.stderr)
+                print("Network function could not read from worker\\n", file=sys.stderr)
     return 0
 
 '''
+
+wq_network_code = \
+'''
+import socket
+import json
+import os
+import sys
+import threading
+import queue
+def remote_execute(func):
+    def remote_wrapper(event, q=None):
+        if q:
```

Comment:
Better to decide to load because of the nature of event, not if q was given, e.g.  `if isinstance(event, str):` and surround with try: except:.

Reply:
Also, is remote_wrapper used anywhere?

Reply:
q is a thread safe data structure that is used to retrieve the result of the function in the case thread execution is used. Since the thread method is being removed with Matt's upcoming PR, this code will be removed there.